feature/google/src/main/java/com/revenuecat/purchases/google/GooglePurchaseOption.kt

```diff
@@ -15,5 +19,7 @@ data class GooglePurchaseOption(
     /**
      * Token used to purchase
      */
-    val token: String
+    val token: String,
```

Comment:
hmmm now some of this data is duplicated in the purchaseInfo 🤔 maybe we could do computed properties for token and purchaseOptionId that return the `purchasingData` values?

Reply:
Ohhh, that's interesting! We might be able to do that 😊

Reply:
Better yet... I'm just going to delete `token` completely from `GooglePurchaseOption` because it doesn't need it for anything anymore 😊